addons/account/models/chart_template.py

```diff
@@ -1,997 +1,735 @@
-             "tax base amount.")
-
-    @api.model
-    def _prepare_transfer_account_template(self, prefix=None):
-        ''' Prepare values to create the transfer account that is an intermediary account used when moving money
-        from a liquidity account to another.
-
-        :return:    A dictionary of values to create a new account.account.
-        '''
-        digits = self.code_digits
-        prefix = prefix or self.transfer_account_code_prefix or ''
-        # Flatten the hierarchy of chart templates.
-        chart_template = self
-        chart_templates = self
-        while chart_template.parent_id:
-            chart_templates += chart_template.parent_id
-            chart_template = chart_template.parent_id
-        new_code = ''
-        for num in range(1, 100):
-            new_code = str(prefix.ljust(digits - 1, '0')) + str(num)
-            rec = self.env['account.account.template'].search(
-                [('code', '=', new_code), ('chart_template_id', 'in', chart_templates.ids)], limit=1)
-            if not rec:
-                break
-        else:
-            raise UserError(_('Cannot generate an unused account code.'))
+    # --------------------------------------------------------------------------------
+    # Template selection
+    # --------------------------------------------------------------------------------
+
+    @lru_cache(None)
+    def get_chart_template_mapping(self):
+        """Get basic information about available CoA and their modules.
+
+        :return: a mapping between the template code and a dictionnary constaining the
+                 name, country id, country name, module dependencies and parent template
+        :rtype: dict[str, dict]
+        """
+        def get_flag(country_code):
+            # get the sequence of 2 regional indicator emoji forming the flag linked to the country code
+            return "".join(chr(int(f"1f1{ord(c)+165:02x}", base=16)) for c in country_code)
+
+        def templ(code, name=None, country='', modules=None, parent=None):
+            country_code = country or code.split('_')[0] if country is not None else None
+            country = country_code and self.ref(f"base.{country_code}", raise_if_not_found=False)
+            country_name = f"{get_flag(country.code)} {country.name}" if country else ''
+            return (code, {
+                'name': country_name and (f"{country_name} - {name}" if name else country_name) or name,
+                'country_id': country and country.id,
+                'country_code': country and country.code,
+                'modules': modules or [f"l10n_{code.split('_')[0]}"],
+                'parent': parent,
+            })
 
-        return {
-            'name': _('Liquidity Transfer'),
-            'code': new_code,
-            'account_type': 'asset_current',
-            'reconcile': True,
-            'chart_template_id': self.id,
-        }
+        return dict([
```

Comment:
It will need to be installed in order to appear in the list by extending `account.chart.template`.
This is just to have an easier selection instead of going through the module installation page, to avoid a few clicks.